.github/workflows/build_jruby.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and upload JRuby runtime
+run-name: "Build and upload JRuby ${{ inputs.jruby_version }}${{ inputs.dry_run && ' (dry run)' || '' }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      jruby_version:
+          description: "The JRuby version to build, specified as X.Y.Z (i.e. 9.4.7.0)"
```

Comment:
Out of curiosity, what happens when a version is given that is already built and uploaded? Are there any guards that prevent this or warnings displayed if this was done by accident?

Reply:
Nope. There are no guards. If you re-build an already built binary, it will overwrite the last one on S3. We're using the `aws s3` command. AFAIK there's no "don't overwrite" but there is an issue from 2017 https://github.com/aws/aws-cli/issues/2874.

Here's the sync help docs from my mac https://gist.github.com/schneems/6c15790fa507c913166ba74604d72b83.

One thought is to add a SHA to the filename once we have manifest files so if there was a bug in the build process the sha would be different and the old one would still work.